frontend/src/components/commons/ProgressPane.jsx

```diff
@@ -23,6 +23,10 @@ const styles = {
 };
 
 class ProgressPane extends Component {
+  constructor(props) {
+    super(props);
+    this.changeNode = this.changeNode.bind(this);
```

Comment:
In ES6, it's preferred to use arrow functions over binding explicitly.

So you can change line 42 to `changeNode = (index) =>` and remove the constructor.

Reply:
fixed in d4d1c7e